Create readme.md

o-andres-m opened the pull request:
Close #1

o-andres-m committed:
Create readme.md

o-andres-m committed:
Update readme.md

jnasselle commented on readme.md:
```diff
@@ -0,0 +1,5 @@
+## Precio_Dolar by Andres
+
+Aplicacion Python para verificar el ultimo precio del dolar actualizado.
+
+Seleccionar desde el menu la opcion deseada, se imprime en pantalla el precio del dolar.
```
Change language to English

o-andres-m committed:
Updates

+ Create readme.md and translate to English
+ Modificate functions in precio_dolar.py, i put the docstring in every one.
+ Change the name of functions and variables to english

o-andres-m commented:
Hi Nicolas, i make the changes. Tell me about that.. Thanks!

jnasselle reviewed (approved):
Nice job @o-andres-m ,

Despite there's a lot of things to improve, the first lesson is to atomize the PRs changes to issue scope. In this case, multiple things were also solved.

New findings will be attacked on new issues
